packages/node_modules/@node-red/editor-client/src/js/ui/palette-editor.js

```diff
@@ -1462,6 +1470,63 @@ RED.palette.editor = (function() {
         })
     }
 
+    const updateStatusWidget = $('<button type="button" class="red-ui-footer-button red-ui-update-status"></button>');
+    let updateAvailable = [];
+
+    function addUpdateInfoToStatusBar() {
+        updateStatusWidget.on("click", function (evt) {
+            RED.actions.invoke("core:manage-palette", {
+                view: "nodes",
+                filter: '"' + updateAvailable.join('", "') + '"'
+            });
+        });
+
+        RED.popover.tooltip(updateStatusWidget, function () {
+            const count = updateAvailable.length || 0;
+            return RED._("palette.editor.updateCount", { count: count });
+        });
+
+        RED.statusBar.add({
+            id: "update",
+            align: "right",
+            element: updateStatusWidget
+        });
+
+        updateStatus({ count: 0 });
+    }
+
+    function refreshUpdateStatus() {
+        updateAvailable = [];
+
+        for (const module of Object.keys(nodeEntries)) {
+            if (loadedIndex.hasOwnProperty(module)) {
+                const moduleInfo = nodeEntries[module].info;
+                if (moduleInfo.pending_version) {
+                    // Module updated
+                    continue;
+                }
+                if (updateAllowed &&
+                    semVerCompare(loadedIndex[module].version, moduleInfo.version) > 0 &&
+                    RED.utils.checkModuleAllowed(module, null, updateAllowList, updateDenyList)
+                ) {
+                    updateAvailable.push(module);
+                }
+            }
+        }
+
+        updateStatus({ count: updateAvailable.length });
+    }
+
+    function updateStatus(opts) {
+        if (opts.count) {
+            RED.statusBar.show("update");
+            updateStatusWidget.empty();
+            $('<span><i class="fa fa-cogs"></i> ' + opts.count + '</span>').appendTo(updateStatusWidget);
```

Comment:
```suggestion
            $('<span><i class="fa fa-cube"></i> ' + opts.count + '</span>').appendTo(updateStatusWidget);
```

We use `fa-cube` as the icon for packages. Suggest using the same here as it relates to the packages.

Reply:
Ok for me, we can use this icon for the update of NR itself